dashboard/test/controllers/code_reviews_controller_test.rb

```diff
@@ -0,0 +1,98 @@
+require 'test_helper'
+
+class CodeReviewsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
+
+  setup_all do
+    @project_owner = create :student
+    @storage_id = create_storage_id_for_user(@project_owner.id)
```

Comment:
Yeah, I'd like to stop stubbing project code and use factories like our other models.  Unfortunately we can't quite do this yet because we don't yet have ActiveRecord models for the projects tables but they will be coming soon.

Reply:
FWIW I stubbed these kinds of calls here, but I think its more complicated than its worth:

https://github.com/code-dot-org/code-dot-org/blob/5a2746b87bee94f0f313788bfcf294b40fd6ae5c/dashboard/test/controllers/code_review_comments_controller_test.rb#L292